### PR TITLE
Ibp 5203 cypress entry details variables

### PR DESCRIPTION
**Pull request by lik-83**
Hi @barbacenasmc, 

In the same test, an input details variable is added to a germplasm list and a value is assigned to it.

### CODE REVIEW

**Review by barbacenasmc — COMMENTED** [review]
Kindly check comments for some selected lines. Thanks!

File: cypress/integration/tests/manage-lists/add-entry-details.feature [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  In order to add entry details to an existing germplasm list
+  As a user
+  I should be able to add entry details to an existing germplasm list
+
+  @smoke-test
+  @setup
+  Scenario: I am on the Germplasm Lists Beta
+    Given I am on the Germplasm Lists Beta page of specified program
+
+  @smoke-test
```
[the remark this answers is not in the document]
Comment (lik-83) [reply cypress/integration/tests/manage-lists/add-entry-details.feature]: Done!

File: cypress/integration/tests/manage-lists/add-entry-details.feature [hunk 2]
```diff
@@ -0,0 +1,22 @@
+@manage-lists
+@add-entry-details
+Feature: Add entry details to an existing list
+  In order to add entry details to an existing germplasm list
+  As a user
+  I should be able to add entry details to an existing germplasm list
+
+  @smoke-test
+  @setup
+  Scenario: I am on the Germplasm Lists Beta
```
[the remark this answers is not in the document]
Comment (lik-83) [reply cypress/integration/tests/manage-lists/add-entry-details.feature]: I implemented the test: https://github.com/IntegratedBreedingPlatform/BMSCypressAutomatedTests/blob/IBP-5203_Cypress_entry_details_variables/cypress/integration/tests/manage-ontology/create-variable.feature

It took me a while to write that test and I didn't have time to implement using the previously created entry details in Manage Ontologies into the entry details section of the germplams list. I need to do that in another iteration.

File: cypress/integration/tests/manage-lists/add-entry-details.feature [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    Given I am on the Germplasm Lists Beta page of specified program
+
+  @smoke-test
+  Scenario: Add entry details and value to an existing germplasm list
+    When I click on a Germplasm List name
```
[the remark this answers is not in the document]
Comment (lik-83) [reply cypress/integration/tests/manage-lists/add-entry-details.feature]: Done!

File: cypress/integration/tests/manage-lists/add-entry-details.feature [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    And I click Add on Entry Details section
+    And I select a Variable
```
[the remark this answers is not in the document]
Comment (lik-83) [reply cypress/integration/tests/manage-lists/add-entry-details.feature]: As I mentioned before, it's still missing to use the entry details created in Manage Ontologies. Also, it's missing to implement an outline scenario for different scale type. I need to do that in another iteration.

File: cypress/integration/tests/manage-lists/add-entry-details.feature [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  As a user
+  I should be able to add entry details to an existing germplasm list
+
+  @smoke-test
+  @setup
+  Scenario: I am on the Germplasm Lists Beta
+    Given I am on the Germplasm Lists Beta page of specified program
+
+  @smoke-test
+  Scenario: Add entry details and value to an existing germplasm list
```
[the remark this answers is not in the document]
Comment (lik-83) [reply cypress/integration/tests/manage-lists/add-entry-details.feature]: As I mentioned before, it's still missing to use the entry details created previously in Manage Ontologies. Also, it's missing to implement an outline for different scale type. I need to do that in another iteration.

File: cypress/integration/tests/manage-lists/add-entry-details.feature (hunk 2 above)
[the remark this answers is not in the document]
Comment (barbacenasmc) [reply cypress/integration/tests/manage-lists/add-entry-details.feature]: Awesome! Noted on this. I will add another subtask for the remaining to-dos

File: cypress/integration/tests/manage-lists/add-entry-details.feature (hunk 5 above)
[the remark this answers is not in the document]
Comment (barbacenasmc) [reply cypress/integration/tests/manage-lists/add-entry-details.feature]: Noted on this.

**Review by barbacenasmc — APPROVED** [review]

**Pull request merged**